packages/tabs/src/Tabs/Tabs.tsx

```diff
@@ -158,35 +156,33 @@ const Tabs = (props: AccessibleTabsProps) => {
   });
 
   return (
-    <LeafyGreenProvider baseFontSize={baseFontSize === 16 ? 16 : 14}>
+    <LeafyGreenProvider
+      baseFontSize={baseFontSize === 16 && size === 'default' ? 16 : 14}
+    >
       <TabDescendantsProvider>
         <TabPanelDescendantProvider>
           <TabsContext.Provider
             value={{
               as,
               darkMode,
               forceRenderAllTabPanels,
-              selectedIndex: selected,
+              selected,
+              size,
             }}
           >
             <div {...rest} className={className} data-lgid={dataLgId}>
-              <div className={tabContainerStyle} id={id}>
+              <div className={tabContainerStyles} id={id}>
                 <div
-                  className={cx(
-                    getListThemeStyles(theme),
-                    tabListElementClassName,
-                  )}
+                  className={getTabListStyles(theme)}
                   data-lgid={LGIDS_TABS.tabList}
                   role="tablist"
                   aria-orientation="horizontal"
                   {...accessibilityProps}
                 >
                   {renderedTabs}
                 </div>
-                <div className={inlineChildrenContainerStyle}>
-                  <div className={inlineChildrenWrapperStyle}>
-                    {inlineChildren}
-                  </div>
+                <div className={inlineChildrenContainerStyles}>
```

Comment:
I found this [PR](https://github.com/mongodb/leafygreen-ui/commit/b708fe19fc2cd2827341335d3d19632bd7f8d63b) related to this change, can you confirm that this doesn't break that?

Reply:
@stephl3 ^^

Reply:
whoops sorry I didn't see the responses on this thread. checking it out now

Reply:
Previously, the bottom-border (added via a linear-gradient background) wasn't extending under the `inlineChildren` but I think that's a bug/incomplete. I've confirmed in both cases where `inlineChildren` are defined that the consumer ends up defining a separate container to extend the border:
1. https://github.com/mongodb/design/blob/c82375b0edb1df241d248538331dc08e8ed72d83/src/app/component/%5Bcomponent%5D/layout.tsx#L86-L96
2. https://github.com/10gen/baas-ui/blob/a9037e5283a31a86b4da7559eeb9a3beea131340/src/common/components/header-tabs/HeaderTabs.tsx#L51-L83

I think it makes sense instead to support this out of the box, so I've moved the border to the div that contains both the tablist and inline children container. I also exported an `inlineChildrenContainerClassName` for consumers to update styling if they need to
| old | new |
| - | - |
| ```
<Tabs
  ...
  inlineChildren={
    <div>
      <button />
      <button />
    </div>
  }
/>
``` | ```
<Tabs
  ...
  inlineChildren={
    <>
      <button />
      <button />
    </>
  }
/>
``` |

Reply:
The icons are no longer center-aligned. 
<img width="970" alt="Screenshot 2024-07-26 at 4 49 23 PM" src="https://github.com/user-attachments/assets/a2fb003f-003b-4870-acf0-b0eb04e5f687">

I'm not sure why we don't extend the border. Maybe it's best to chat with the team; perhaps someone else might have more insight.
Also, If we do make this change, this would be a `major` change

Reply:
Ahh right, I thought it wouldn't have to be a breaking change by not touching the `align-items` property. I updated the `align-items: ~~stretch~~center;` which will work in the story but will break all existing implementations because they define a border in their application code. They should be okay to just drop their container code in favor of the one defined in the LG tabs component